.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn pre-commit
```

Comment:
`husky` configures a `git` hook for `pre-commit` that will run this file anytime a contributor runs `git commit`. This hook runs `yarn pre-commit`, which executes the `pre-commit` script defined in `package.json`.